rust/src/modbus/modbus.rs

```diff
@@ -183,8 +182,9 @@ impl ModbusState {
 
     pub fn parse(&mut self, input: &[u8], direction: Direction) -> AppLayerResult {
         let mut rest = input;
+        let modbus_parser = sawp_modbus::Modbus::default();
```

Comment:
The `probe_strict` option is off by default and I think you intend to enable it. However, looking at [the source](https://docs.rs/sawp-modbus/0.10.0/src/sawp_modbus/lib.rs.html#471), I forgot to make the field public. I'll release a patch for that.